src/config/shared.js

```diff
@@ -32,3 +32,20 @@ export function extractBucketMetadata(transformedData) {
   etc.bucketSortOrder = sorter;
   return etc;
 }
+
+export default {
+  activeBuckets: {
```

Comment:
shouldn't `reference` also be here in some way? It doesn't look like the reference point persists now.